app.js

```diff
@@ -262,6 +354,32 @@ app.get('/hubs/:hubSlug/activities', function(req, res){
   }
 })
 
+app.get('/hubs/:hubSlug/devices', function(req, res){
+  hubSlug = req.params.hubSlug
+  harmonyHubClient = harmonyHubClients[hubSlug]
+
+  if (harmonyHubClient) {
+    var devices = [];
+    res.json({devices: cachedHarmonyDevices(hubSlug).map(function(device) {
```

Comment:
Ah, yeah, that's probably what I was going for. I bounced back and forth from this project to others like a dozen times, so it was hard to keep track of what I was doing.